README.md

```diff
@@ -216,7 +224,7 @@ Since our initial `0.0.1` version of the package, many modifications have been m
       link: http://fake-link
 ```
 
-Run `elastic-build` again and the upload command. The `0.1.0` version of the package will be updated to version 0.2.0. Only 1 version of the same package can be installed at the same time. But like this different versions of a package can be rolled out over time.
+Run `elastic-package check` again and the upload command. The `0.1.0` version of the package will be updated to version 0.2.0. Only 1 version of the same package can be installed at the same time. But like this different versions of a package can be rolled out over time.
```

Comment:
I was thinking to rephrase the first sentence, what about one of these ?
- Run `elastic-package check` again and upload the package.
- Run `elastic-package check` again and the install command.

Reply:
I went for: "Run `elastic-package check` again and the `elastic-package install` command"